snapcraft/internal/meta.py

```diff
@@ -0,0 +1,259 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
It is just like `states` (although a package); as long as it is in `internal` it should be fine.